### PR TITLE
Support updates

### PR DESCRIPTION
**Pull request by andrewawagner**

### CODE REVIEW

File: contact.md [hunk 1]
```diff
@@ -9,14 +9,28 @@ summary: Need to contact Catalyze? We're always here to help!
 
 For product inquiries, please contact [Sales](mailto:sales@catalyze.io)!
 
-## Support
+## Stratum Support
 
-Contact Catalyze Support by using this [form](https://catalyzeio.zendesk.com/hc/en-us/requests/new).  For requests regarding PaaS, please include your environment id. You can find the environment id in the URL of the Dashboard for your environment.  You can also find the environment id when using the [paas-cli ](https://github.com/catalyzeio/cli) with the command `catalyze environments`
+Existing customers should contact Catalyze Support by logging into https://product.catalyze.io/stratum and clicking on "Contact Support" for the environment with which they need support.
```
Comment (jault3) [inline contact.md]: Make that a link

**Comment by jault3** [comment]
Other than the one link, LGTM

**Comment by rr1000** [comment]
LGTM

**Comment by forana** [comment]
closing to merge https://github.com/catalyzeio/stratum-docs/pull/32 instead

**Pull request closed without merging**